Create daily-scrum-meeting.md (#122)
atolye15/checklist: checklists/daily-scrum-meeting-.md

---
title: Daily Scrum Meeting
category: Project Management
date: "2020-03-21"
tags: ['scrum', 'agile']
description: Go agile! 🚀 Daily Scrum Meetings are like a good-brewed coffee, don't forget it when you start a new work-day.
---

> - [ ] At fist, set a specific time and place for the daily meeting  

Reviewer: ```suggestion
- [ ] At first, set a specific time and place for the daily meeting  
```

So that, it will be easier for the team to gain habit.

- [ ] Check the time  
We prefer to do daily scrum meeting in the morning. So that everyone knows what will be done today. 

- [ ] Start to call your team  
Team will gain habit over time. You can use some clues while calling them, like the key of the project. Remind them it is daily time! 

- [ ] Remain standing!  

> This way, meeting will(and should) last maximum 15 min.  

Reviewer: ```suggestion
This way, meeting will(and should) last maximum of 15 mins.  
```

> - [ ] Ensure everyone in the team is here

Reviewer: ```suggestion
- [ ] Ensure everyone in the team is on the meeting
```

> - [ ] Ask the fisrt question, "What did we do yesterday?"  

Reviewer: ```suggestion
- [ ] Ask the first question, "What did we do yesterday?"  
```

Give a chance to everyone to speak.

- [ ] Learn the answer of "What are we doing today?"  

> Again, everyone must be give an answer.

Reviewer: ```suggestion
Again, everyone should give an answer.
```
